dotCMS/src/main/java/com/dotcms/services/VanityUrlServices.java

```diff
@@ -0,0 +1,187 @@
+package com.dotcms.services;
+
+import com.dotcms.cache.VanityUrlCache;
+import com.dotcms.util.VanityUrlUtil;
+import com.dotcms.vanityurl.business.VanityUrlAPI;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.vanityurl.business.VanityUrlAPI'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)